app/models/manageiq/providers/amazon/cloud_manager/vm_or_template_shared/scanning.rb

```diff
@@ -24,21 +26,130 @@ def perform_metadata_scan(ost)
     @s3                    = ext_management_system.connect(connect_args)
     raise "Unable to obtain a new S3 resource" unless @s3
     ssaq_args                 = {}
-    ssaq_args[:ssa_bucket]    = name
+    ssaq_args[:ssa_bucket]    = "evm-prototype"
```

Comment:
Uh- yes.  I thought I actually removed that before pushing.   But we should probably come up with a game plan for the naming before going forward with the PR...